src/controllers/aer_controller.hpp

```diff
@@ -922,7 +922,7 @@ Result Controller::execute(std::vector<Circuit> &circuits,
   // Execute each circuit in a try block
   try {
     //check if multi-chunk distribution is required
-    bool multi_chunk_required_ = false;
+    multi_chunk_required_ = false;
```

Comment:
@doichanj is this ok? `multi_chunk_required_` is a member of `Controller`.

Reply:
Yes, bool is not needed, thank you